### PR TITLE
fix: add dummy data and fix files of the model based on the changes made on the DB

### PR DESCRIPTION
**Pull request by darwinTtito**

### CODE REVIEW

File: pkg/model/complement/awardModel.go [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-	Description string `json:"description"`
-	Event       string `json:"event"`
-	Place       int32  `json:"place"`
-	IsVerified  bool   `json:"is_verified"`
-	DocVerifier []byte `json:"doc_verifier"`
-	CareerId    uint64 `json:"career_id"`
-	AlbumId     uint64 `json:"album_id"`
+	Id               uint64    `json:"id"`
+	Year             string    `json:"year"`
+	Period           string    `json:"period"`
```
Comment (PDFAtauchi) [inline pkg/model/complement/awardModel.go]: change Period "type string" -> "type uint32"

File: pkg/model/course/courseModel.go [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+	Period           int32     `json:"period"`
+	Semester         int32     `json:"semester"`
+	TotalHours       int32     `json:"total_hours"`
+	IsVerified       int32     `json:"is_verified"`
+	DocVerifier      []byte    `json:"doc_verifier"`
+	CreatedDate      time.Time `json:"created_date"`
+	LastModifiedDate time.Time `json:"last_modified_date"`
+	TypeCourseId     uint64    `json:"type_course_id"`
+	CareerId         uint64    `json:"carrera_id"`
+	Professor        uint64    `json:"professor"`
```
Comment (PDFAtauchi) [inline pkg/model/course/courseModel.go]: change Professor -> ProfessorId

**Review by PDFAtauchi — CHANGES_REQUESTED** [review]

**Review by Kleiber — APPROVED** [review]

File: pkg/model/complement/awardModel.go (hunk 1 above)
Comment (darwinTtito) [reply pkg/model/complement/awardModel.go]: Done!

File: pkg/model/course/courseModel.go (hunk 2 above)
Comment (darwinTtito) [reply pkg/model/course/courseModel.go]: Done!

**Review by PDFAtauchi — APPROVED** [review]

**Pull request merged**